cmd/mysterium_server/command_run/options.go

```diff
@@ -10,6 +10,7 @@ type CommandOptions struct {
 	DirectoryConfig   string
 	DirectoryRuntime  string
 	DirectoryKeystore string
+	Passphrase        string
```

Comment:
`passphrase` -> `password` to be consistent with previous naming (i think in Tequilapi.CreateIdentity endpoint).
Or sync both places

Reply:
https://mysteriumnetwork.atlassian.net/browse/MYST-261